### ISSUE TITLE
Negative globs do not work when a `cwd` option is passed

### ISSUE THREAD

**Issue by JGets** [gulpjs/glob-watcher#46]
When a `cwd` option is provided, it appears as though negative globs do not properly get applied.

For example, given a watcher:
```javascript
watch([ 'bar/**', '!bar/*.css' ], { cwd : 'foo' }, someFunction);
```
A change to `foo/bar/styles.css` still triggers `someFunction` to be invoked.

**Comment by JGets** [comment]
I think this is due to `chokidar` prepending the `cwd` to the path before passing it to the [`shouldBeIgnored()`](https://github.com/gulpjs/glob-watcher/blob/master/index.js#L74) `ignored` function, where it is then compared against the globs without the `cwd` applied.

**Comment by phated** [comment]
@JGets Thanks for reporting. Sorry for the extreme delay here. The team has been swamped and I had to create a repro for this in our test suite.

I'm getting a patch published as 5.0.4 once CI finishes.
